lib/puppet/type/selboolean.rb

```diff
@@ -9,9 +9,23 @@ module Puppet
     end
 
     newproperty(:value) do
-      desc "Whether the SELinux boolean should be enabled or disabled."
-      newvalue(:on)
-      newvalue(:off)
+      VALUE_DOC="Valid values are: \"on\"/\"true\"/\"off\"/\"false\""
```

Comment:
Would be better to use a local variable `value_doc` instead of the constant `VALUE_DOC` because in ruby the constants are shared and can be accessed and modified from anywhere.